### PR TITLE
Add support for retry on MS rate limit

### PR DESCRIPTION
**Pull request by ostolero**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3843)

## Description
Add support for retry on requests where 429 Rate limit error is recieved

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml [hunk 1]
```diff
@@ -9,30 +9,32 @@ configuration:
   required: true
```
Comment (JudahSchwartz) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml:9]: polling : true 
should be added to these commands, as well as the first_run_flag argument

**Review by JudahSchwartz — COMMENTED** [review]
Looks good! Lets make sure we test the polling scenario, and add the fields needed to the yml

**Review by JudahSchwartz — APPROVED** [review]
Release notes are not correct. We know which integrations support it. The other integrations should say "Internal code improvements" or something similar, not indicate that they support dealing with rate limiting

**Comment by JudahSchwartz** [comment]
@GuyAfik please review the change made here to graph? Had to merge here

**Review by GuyAfik — APPROVED** [review]

**Pull request merged**